src/settings.rs

```diff
@@ -103,6 +103,12 @@ pub struct Settings {
     /// Proximity precision settings.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub proximity_precision: Option<String>,
+    /// Fields in which to search for matching query words sorted by order of importance.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub separator_tokens: Option<Vec<String>>,
+    /// Fields in which to search for matching query words sorted by order of importance.
```

Comment:
Please take a look at the documentation for these comments; https://www.meilisearch.com/docs/reference/api/settings#non-separator-tokens

```suggestion
    /// Remove tokens from Meilisearch's default [list of word separators](https://www.meilisearch.com/docs/learn/engine/datatypes#string).
```